### ISSUE TITLE
Create Darwin Core of ICE2011 phytoplankton microplankton & handnet

### ISSUE THREAD

**Issue by cnrdh** [npolar/marine-db#53]
Notice: ICE2011 is not part of the IOPAN corpus, but was counted by Philipp Assmy.

**Comment by cnrdh** [comment]
I propose to take ICE2011 out of the IOPAN milestone, and rather see if AW or PA can transpose the original data.

**Comment by AnetteWold** [comment]
I assume you want the data transformed to lomng format similar to that used in google docs. I can do it but it might take me some time

**Comment by cnrdh** [comment]
Yes, that's what I suggest. I'll be happy to help out once it's in long format.
